exec/build-commands/src/approveBuilds.ts

```diff
@@ -143,6 +146,9 @@ Do you approve?`,
       ...opts,
       onlyBuiltDependencies: updatedOnlyBuiltDependencies,
     }, buildPackages)
+  } else if (modulesManifest) {
+    delete modulesManifest.ignoredBuilds
+    await writeModulesManifest(modulesDir, modulesManifest as StrictModules)
   }
```

Comment:
Originally, I thought the bug was because `rebuild` didn't call `writeModulesManifest`, but it turns out that `rebuild` wasn't called at all.

Although, this solution seems a bit hacky, maybe we should just remove `if (buildPackages.length)` instead?